client/tenant/client_tenantadm.go

```diff
@@ -145,6 +148,8 @@ func (c *Client) GetTenant(ctx context.Context, username string, client apiclien
 
 	rsp, err := client.Do(req.WithContext(ctx))
 	if err != nil {
+		repl := strings.NewReplacer(username, "REDACTED", usernameQ, "REDACTED")
+		err = errors.New(repl.Replace(err.Error()))
```

Comment:
same here, consider `ReplaceAll`?

Reply:
`ReplaceAll` does not support multiple replacement pairs in one call.